lib/oli/accounts.ex

```diff
@@ -420,9 +421,8 @@ defmodule Oli.Accounts do
   @doc """
   Searches for a list of authors with an email matching a wildcard pattern
   """
-  def search_authors_matching(query) do
-    q = query
-    q = "%" <> q <> "%"
+  def search_authors_matching(query, exact_match \\ false) do
```

Comment:
@gastonabella are there other places that this function is used in the app? If that was the only spot - wonder if it makes sense to make it always an exact match since it was a security issue. If it is used elsewhere - disregard comment.

Reply:
There aren't, but I left it initially just in case. Will do the change, thanks!